response_operations_ui/views/respondents.py

```diff
@@ -23,12 +23,12 @@ def respondent_search():
     if form.validate_on_submit():
         email = request.form.get('query')
 
-        respondent = respondent_controllers.search_respondent_by_email(email)
+        respondent = party_controller.search_respondent_by_email(email)
 
-        if respondent.get('id'):
-            return redirect(url_for('respondent_bp.respondent_details', respondent_id=respondent['id']))
+        if respondent is None:
+            response = 'No Respondent found for ' + email
         else:
-            response = respondent['Response']
+            return redirect(url_for('respondent_bp.respondent_details', respondent_id=respondent['id']))
```

Comment:
Since there's a return in the else we can replace this with just an if statement
```
if respondent:
    return redirect(url_for(...))
response = 'No Respondent found for ' + email
return render_template('search-respondent.html', response=response, form=form, breadcrumbs=breadcrumbs)
```